jarvis.gemspec

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'jarvis/version'
+
+Gem::Specification.new do |spec|
+  spec.platform    = Gem::Platform::RUBY
+  spec.name        = 'jarvis'
+  spec.version     = Jarvis.version
+  spec.authors     = ['Thomas Burgaud']
+  spec.email       = ['thomas.burgaud@cashbee.fr']
+  spec.summary     = ''
+  spec.description = ''
```

Comment:
mention this gem is "made by cashbee" here ?

Reply:
no let's put this in the readme.md

Reply:
https://github.com/cashbee-team/jarvis/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R41